keep the same:
	maintenance set python auto-load
	maintenance show python auto-load

this one has different output
	maintenance print section-scripts
as replaced by
	info auto-load python-scripts

**NOTE**: This patch is deprecated (but still applied).  See patch
`gdb-rhbz921685-maint-set-python-print-stack.patch' for more details.

Index: gdb-7.2/gdb/python/py-auto-load.c
===================================================================
--- gdb-7.2.orig/gdb/python/py-auto-load.c
+++ gdb-7.2/gdb/python/py-auto-load.c
@@ -242,6 +242,22 @@ info_auto_load_python_scripts (char *pat
 {
   auto_load_info_scripts (pattern, from_tty, &script_language_python);
 }
+
+/* RHEL - Function for use by 'maint set python' prefix command.  */
+
+static void
+set_python (char *args, int from_tty)
+{
+  help_list (set_python_list, "maintenance set python ", -1, gdb_stdout);
+}
+
+/* RHEL - Function for use by 'maint show python' prefix command.  */
+
+static void
+show_python (char *args, int from_tty)
+{
+  cmd_show_list (show_python_list, from_tty, "");
+}
 
 void
 gdbpy_initialize_auto_load (void)
@@ -286,6 +302,36 @@ Usage: info auto-load python-scripts [RE
   cmd = add_info ("auto-load-scripts", info_auto_load_python_scripts, _("\
 Print the list of automatically loaded Python scripts, deprecated."));
   deprecate_cmd (cmd, "info auto-load python-scripts");
+
+  /* RHEL backward compatibility.  */
+  {
+    /* Lists for 'maint set python' commands.  */
+    static struct cmd_list_element *set_python_list;
+    static struct cmd_list_element *show_python_list;
+
+    add_prefix_cmd ("python", no_class, show_python,
+		    _("Prefix command for python maintenance settings."),
+		    &show_python_list, "maintenance show python ", 0,
+		    &maintenance_show_cmdlist);
+    add_prefix_cmd ("python", no_class, set_python,
+		    _("Prefix command for python maintenance settings."),
+		    &set_python_list, "maintenance set python ", 0,
+		    &maintenance_set_cmdlist);
+
+    add_setshow_boolean_cmd ("auto-load", class_maintenance,
+			     &auto_load_python_scripts, _("\
+Enable or disable auto-loading of Python code when an object is opened."), _("\
+Show whether Python code will be auto-loaded when an object is opened."), _("\
+Enables or disables auto-loading of Python code when an object is opened."),
+			     NULL, NULL,
+			     &set_python_list,
+			     &show_python_list);
+
+    /* The output is not fully backward compatible.  */
+    add_cmd ("section-scripts", class_maintenance, info_auto_load_python_scripts,
+	     _("Print dump of auto-loaded section scripts matching REGEXP."),
+	     &maintenanceprintlist);
+  }
 }
 
 #else /* ! HAVE_PYTHON */
